### PR TITLE
fix: replicate mesh and material over network

### PR DESCRIPTION
**Pull request by meetcshah19**
### Contains

Fix - Replicates mesh and material of Skeletal Mesh Component over the network.

### How to test

- Play Josharias Survival in multiplayer mode with the latest Manual Labor, with 2 players.
- Give yourself a crude shear using ```give crudeshears``` and spawn a sheep using ```spawnPrefab sheep```.
- Use shear on sheep and check if the model change is replicated for the second player

### CODE REVIEW

File: engine/src/main/java/org/terasology/engine/rendering/logic/SkeletalMeshComponent.java [hunk 1]
```diff
@@ -20,7 +20,10 @@
 
 @ForceBlockActive
 public class SkeletalMeshComponent implements VisualComponent {
+    @Replicate
     public SkeletalMesh mesh;
+
+    @Replicate
     public Material material;
```
Comment (pollend) [inline engine/src/main/java/org/terasology/engine/rendering/logic/SkeletalMeshComponent.java:28]: I don't know about straight replicating this component.

**Comment by pollend** [comment]
Not sure about the implications this has on networking. We have problems with the current system. Might exasperate there problem if we do this.

**Comment by jdrueckert** [comment]
If this would be a feature, I would agree... However this is a _fix_.
If this makes problems with the networking more present, that's unfortunate and we have to try to improve on our networking, but this _is_ a desired change to enable proper multiplayer behaviour.

**Review by jdrueckert — APPROVED** [review]
Tests out fine :+1:

**Pull request merged**